src/content/basics/azure-architecture/index.md

```diff
@@ -35,10 +35,12 @@ One is the Kubernetes API that can be connected to your Identity Management Syst
 
 ![Service Architecture](/img/architecture-azure-services.png)
 
-To make your life easier, we have developed a lot of different services within our Control Plane that allow both our operations team and you as users of our API and interfaces to easily manage Kubernetes clusters. Most of these services should be self explanatory. 
+To make your life easier, we have developed a lot of different services within our Control Plane that allow both our operations team and you as, users of our API and interfaces to easily manage Kubernetes clusters. Most of these services should be self explanatory, due to the level of granularity we maintain. 
```

Comment:
Sorry but the addition of `due to the level of granularity we maintain.` makes this less clear for me.

I think the meaning about being self explanatory is because the terms used are common in this space.

Reply:
The thing is that the existing text says 'should be self-explanatory' and I want to add the reasoning as to why it's self-explanatory, because what happens if an individual does not feel it is indeed self-explanatory. 
The addition was my attempt to allude to the fact that we keep a separation of concerns, so each operator does one thing, which is why it is self-explanatory (Whew, that was long. Maybe we need to iron this out verbally).

Reply:
OK that makes sense but if the audience is other architects they should understand those terms.

But I'm also fine with just removing self explanatory. @piontec you have fresher eyes than me. WDYT?